webext permissions: activeTab and menu for a subframe (#27098)
mdn/content: files/en-us/mozilla/add-ons/webextensions/manifest.json/permissions/index.md

> **Note:** You can only get access to the tab/data that was there, when the user interaction occurred (e.g. the click). When the active tab navigates away (e.g., due to finishing loading or some other event), the permission does not grant you access to the tab anymore.

> [Context menu](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Context_menu_items) opened inside a subframe gives permissions to inject content scripts or styles in this particular frame, not to the top page of the tab. Behavior of older Firefox versions, e.g. 102 ESR, is different however and either specific host permission or `<all_urls>` is necessary in addition to execute content scripts in the `frameId` passed to {{webextAPIref("menus/onClicked", "browser.menus.onClicked")}} event listener.

Reviewer: ```suggestion
A [context menu](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Context_menu_items) opened inside a subframe gives permissions to inject content scripts or styles to that subframe, but not to the top page of the tab. Behavior of older Firefox versions (for example, 102 ESR) is different however and requires either specific host permission or `<all_urls>` to execute content scripts in the `frameId` passed to the {{webextAPIref("menus/onClicked", "browser.menus.onClicked")}} event listener.
```

Author: @jpmedley, thank you for review. In general I agree with your corrections, however instead of

> Behavior … is different however and requires … permission …

I would consider

> Behavior … is different however and **the extension** requires … permissions …
